test-cli.py

```diff
@@ -97,20 +97,20 @@ def get_uuid(line):
     return m.group(1)
 
 
-def sanitize(string):
+def sanitize(string, max_chars=256):
     try:
         string = string.decode('utf-8')
-        n = 256
-        if len(string) > n:
-            string = string[:n] + ' (...more...)'
+        if len(string) > max_chars:
+            string = string[:max_chars] + ' (...more...)'
         return string
     except UnicodeDecodeError:
         return '<binary>\n'
 
 
-def run_command(args, expected_exit_code=0):
-    # Decode args into ASCII as travis can't handle them otherwise
-    args = [arg.encode('ascii', 'ignore') for arg in args]
+def run_command(args, expected_exit_code=0, max_output_chars=256):
```

Comment:
What happened here? If the commented out line is no longer relevant (and the comment statement is no longer correct), please remove them.